node/sc/bridge_manager.go

```diff
@@ -211,7 +222,7 @@ func (bi *BridgeInfo) processingPendingRequestEvents() error {
 	logger.Trace("Get ready request value transfer event", "len(readyEvent)", len(ReadyEvent), "len(pendingEvent)", bi.pendingRequestEvent.Len())
 
 	for idx, ev := range ReadyEvent {
-		if ev.RequestNonce < bi.lowerHandleNonce {
+		if ev.RequestNonce < bi.lowerHandleNonce || bi.handledEvent.Exist(ev.RequestNonce) {
```

Comment:
This seems to be less computational cost
```suggestion
		if ev.RequestNonce <= bi.lowerHandleNonce || bi.handledEvent.Exist(ev.RequestNonce) {
```

Reply:
`lowerHandleNonce` may not be handled. So we should not ignore it.